frontend/src/scss/global.scss

```diff
@@ -50,3 +50,13 @@ body {font-feature-settings: "ss03", "calt", "rlig", "kern", "cv05", "cv07", "cv
 .flex-full {
 	flex: 1 1 0;
 }
+
+.v-sheet > .v-expansion-panels,
+.v-card > .v-expansion-panels {
+	border-top: 1px solid #eee;
+	border-bottom: 1px solid #eee;
+
+	> .v-expansion-panel + .v-expansion-panel {
+		border-top: 1px solid #eee;
+	}
+}
```

Comment:
If you restyle this, also use the `accordion` mode on the `v-expansion-panels` so that the panels don't move down when opening.

Reply:
This is not fixed yet: 
![moving-line](https://user-images.githubusercontent.com/7566995/102724835-c8aa2c00-4312-11eb-9cee-b232caa0d1ca.gif)
Could you remove the additional top margin that the expansion panel has in the open state? That way it doesn't look as if the top period grows when I open the bottom period.